### DISCUSSION TITLE
Installation help!!!

### DISCUSSION THREAD

**Discussion by ormax56** [PhlexPlexico/G5API#110] [Help]
Would you be so kind as to write a manual for beginners, because not all specialists are in Linux. I do everything according to the manual, but one thing does not work, then something else, and not everything is clear. When I installed Ranked-stats from niekcandaele. that everything turned out in 15 minutes, everything is clear and understandable there.
The sequence of actions itself is unclear.

**Comment by ormax56** [comment]


**Reply by PhlexPlexico** [reply]
Just FYI the wiki has some information about the database stuff here https://github.com/PhlexPlexico/G5API/wiki/Configuration#apply-database-configs

As for the `yarn install` it shouldn't need `install` or the `--production` flag I believe? `yarn` should just install all the packages.
